crates/lang_srv/Cargo.toml

```diff
@@ -7,7 +7,12 @@ edition = "2021"
 name = "roc_ls"
 path = "src/server.rs"
 
+[dev-dependencies]
+expect-test = "1.4.1"
```

Comment:
Just curious, is there a reason you prefer this over `insta`? We use insta in other parts of the project. If there is no preference, no need to change this now; just curious.

Reply:
Oh, I didn't realise that you used instead elsewhere. I switched it because insta seemed overkill and I preferred the simplicity of just embedding everything in the file and not having an external program to run. I'm happy to switch it if you'd like to keep things consistent.

Reply:
No, let's keep it for now (so we can merge!). I'll probably replace it, but we can save it for later.